<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Jurnal</class>
 <widget class="QWidget" name="Jurnal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jurnal App</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="citat"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="intrare_jurnal"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="save_btn">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="load_btn">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="Citat_label">
     <property name="text">
      <string>Citat:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="Jurnal_label">
     <property name="text">
      <string>Jurnal:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
